HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,15 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: addr
+Returns the ip and TCP/UDP port being used by the specified server
```

Comment:
It's necessary to specify that this is a NetAddr instance. Suggest: "Returns the network address of the server as a NetAddr object."